operator/v1/types_network.go

```diff
@@ -60,6 +60,11 @@ type NetworkSpec struct {
 	// 'false' and multiple network support is enabled.
 	DisableMultiNetwork *bool `json:"disableMultiNetwork,omitempty"`
 
+	// useMultiNetworkPolicy enables a controller which allows for
+	// NetworkPolicy-like objects to be used on additional networks as
+	// created by Multus CNI
```

Comment:
How about:

useMultiNetworkPolicy enables a controller which allows for NetworkPolicy-like objects to be used on additional networks as created by Multus CNI.  If unset, this property defaults  to 'false' and multi-network NetworkPolicy-like objects are ignored.  If 'disableMultiNetwork' is 'true' then the value of this field is ignored.

Reply:
Thanks, changed.